### DISCUSSION TITLE
thanhle.blog/thuat-ngu-trong-frontend-optimization

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#68] [Ideas]
# thanhle.blog/thuat-ngu-trong-frontend-optimization

Cùng tìm hiểu xem các keyword khi mình làm frontend optimization thì có những gì nhé

https://thanhle.blog/blog/thuat-ngu-trong-frontend-optimization

**Comment by chungnq253** [comment]
Bài viết rất cô đọng Thành ơi, mỗi tội video lỗi r 😂

**Reply by thanhlmm** [reply]
Thanks bro, để vào hotfix :))
